CONTRIBUTING.rst

```diff
@@ -129,7 +129,7 @@ Running Tests
   *Using a Development Checkout* above).
 
 - To run the full set of gcloud-python tests on all platforms, install ``tox``
-  (http://codespeak.net/~hpk/tox/) into a system Python.  The ``tox`` console
+  (https://tox.readthedocs.org) into a system Python.  The ``tox`` console
```

Comment:
Hmm, per https://pypi.python.org/pypi/tox, the "homepage" is https://testrun.org/tox/ (redirects to https://testrun.org/tox/latest/), which is likely a CNAME pointing to the RTD site.  I think we'd be better off echoing that, here.